content/influxdb/cloud-dedicated/admin/sso.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Set up and use single sign-on (SSO)
+description:
+  Set up and use single sign-on (SSO) to authenicate access to your InfluxDB Cluster.
+menu:
+  influxdb_cloud_dedicated:
+    name: Set up and use SSO
+    parent: Administer InfluxDB Cloud
+weight: 106
+---
+
+{{< product-name >}} supports single sign-on (SSO) integrations through the use
+of [Auth0](https://auth0.com) and your identity provider of choice.
+Use SSO to provide users seamless access to your {{< product-name >}} cluster
+with an existing set of credentials. 
+
+{{% cloud %}}
+#### Contact InfluxData sales to enable SSO
+
+SSO is a paid upgrade to your {{< product-name >}} cluster.
+To begin the process of enabling SSO, contact InfluxData Sales:
+
+<a class="btn" href="https://www.influxdata.com/contact-sales/">Contact InfluxData Sales</a>
+{{% /cloud %}}
+
+## SSO authorization flow
+
+With SSO enabled, whenever a user attempts to log into your {{< product-name >}}
+cluster, the following occurs:
+
+1.  InfluxDB sends an authentication request to the InfluxData-managed Auth0 service.
+2.  Auth0 sends the provided credentials to your identity provider.
+3.  Your identity provider grants or denies authorization based on the provided
+    credentials and returns the appropriate response to Auth0.
+4.  Auth0 returns the authorization response to {{< product-name >}} which grants
+    or denies access to the user.
+
+{{< html-diagram/sso-auth-flow >}}
+
+## Set up your identity provider
+
+For information about setting up and configuring your identity provider, refer
+to your identity provider's documentation.
+You can use any identity provider **supported by Auth0**:
+
+- [Social identity providers supported by Auth0 {{< icon "export" >}}](https://auth0.com/docs/authenticate/identity-providers/social-identity-providers)
+- [Enterprise identity providers supported by Auth0 {{< icon "export" >}}](https://auth0.com/docs/authenticate/identity-providers/enterprise-identity-providers)
+- [Legal identity providers supported by Auth0 {{< icon "export" >}}](https://auth0.com/docs/authenticate/identity-providers/enterprise-identity-providers)
+
+## Connect your identity provider to Auth0
+
+To integrate your identity provider with the InfluxData-managed Auth0 service:
+
+1.  **Create a new application or client** in your identity provider to use with
+    Auth0 and your {{< product-name >}} cluster.
+
+2.  **Provide the necessary connection credentials to InfluxData support**.
+    What credentials are needed depends on your identity provider and the
+    protocol you're using. For example:
+
+    | Protocol | Required credentials          |
+    | :------- | :---------------------------- |
+    | **OIDC** | Client secret                 |
+    | **SAML** | Identity provider certificate |
```

Comment:
@jlytle-influxdata You had mentioned that the support team has boilerplates for each of the providers/protocols. Would it be valuable to list the known details required for each protocol here? Or is it too context-specific?

Reply:
> @jlytle-influxdata You had mentioned that the support team has boilerplates for each of the providers/protocols. Would it be valuable to list the known details required for each protocol here? Or is it too context-specific?

OIDC and SAML are far and away the most commonly used and what you have above will account for the differences. There may be other differences between IdPs that we have not encountered yet.

I may have mis-assumed that our team built the boilerplates, they may just be entrypoints that Auth0 provides - a screenshot so you know what it looks like - some of these do not have any configured connections because we haven't had a customer using them yet:
![image](https://github.com/influxdata/docs-v2/assets/98980787/be80c9f5-15b9-42b6-8f05-a27c66be336d)

Reply:
Another note on the difference between OIDC and SAML:
The certificate used for SAML is public and therefore does not require secure transmission to us.
The Client Secret used for OIDC needs to be transmitted securely, they cannot provide it in the form I supply to them to fill out. We have been using 1password shared vaults and inviting customers as guests so they can submit the client secret securely.